code/src/main/kotlin/com/expediagroup/sdk/core/logging/masking/MaskLogsUtils.kt

```diff
@@ -0,0 +1,100 @@
+package com.expediagroup.sdk.core.logging.masking
+
+import com.ebay.ejmask.core.BaseFilter
+import com.ebay.ejmask.core.EJMask
+import com.ebay.ejmask.core.EJMaskInitializer
+import com.ebay.ejmask.core.util.LoggerUtil
+
+/**
+ * Masks sensitive information within the provided log string.
+ *
+ * @param logs The log string that may contain sensitive information requiring masking.
+ * @return A new log string with sensitive information masked.
+ */
+fun maskLogs(logs: String): String {
```

Comment:
Do we use these utils anywhere?

Reply:
not yet, @OmarAlJarrah was working on log masking redesign. I left these as a reference. Do you think we should remove them?